now.json

```diff
@@ -1,23 +1,9 @@
 {
   "name": "frontity-services",
-  "version": 2,
+  "version": 1,
   "alias": "services.frontity.cloud",
   "regions": ["all"],
```

Comment:
Can you remove `regions` and add `scale` here as well? Same settings.

Reply:
Oh yes, I missed that.